lib/kamal/configuration/accessory.rb

```diff
@@ -1,6 +1,8 @@
 class Kamal::Configuration::Accessory
   include Kamal::Configuration::Validation
 
+  NETWORK = "kamal"
```

Comment:
I think this constant should be stored at `Kamal::Configuration::DEFAULT_NETWORK`

Reply:
Done.